package.json

```diff
@@ -63,26 +64,33 @@
     "@semantic-release/changelog": "^6.0.1",
     "@semantic-release/git": "^10.0.1",
     "@semantic-release/npm": "^9.0.1",
-    "@trivago/prettier-plugin-sort-imports": "3.4.0",
+    "@trivago/prettier-plugin-sort-imports": "^4.0.0",
     "@typechain/ethers-v5": "^7.2.0",
     "@typechain/hardhat": "^2.3.1",
-    "@types/chai": "^4.3.1",
-    "@types/mocha": "^9.1.1",
-    "@types/node": "^12.20.50",
-    "@typescript-eslint/eslint-plugin": "^5.27.1",
-    "@typescript-eslint/parser": "^5.27.1",
+    "@types/chai": "^4.3.4",
+    "@types/fs-extra": "^9.0.13",
+    "@types/mocha": "^10.0.0",
+    "@types/node": "^18.16.3",
+    "@typescript-eslint/eslint-plugin": "^5.44.0",
+    "@typescript-eslint/parser": "^5.44.0",
     "@venusprotocol/governance-contracts": "^1.0.0",
-    "@venusprotocol/oracle": "1.7.2",
-    "@venusprotocol/venus-protocol": "3.0.0-dev.18",
-    "bignumber.js": "9.0.0",
-    "chai": "^4.3.6",
-    "dotenv": "^10.0.0",
-    "eslint": "^7.32.0",
-    "eslint-config-prettier": "8.5.0",
+    "@venusprotocol/isolated-pools": "2.3.0-dev.7",
+    "@venusprotocol/oracle": "1.8.0-dev.7",
+    "@venusprotocol/venus-protocol": "6.1.0-dev.6",
```

Comment:
It can be `^7.0.0`

Reply:
Done https://github.com/VenusProtocol/protocol-reserve/pull/9/commits/2339328efb9571e2be2f7b09fd0fd6aad0798f07